fidesctl/src/fidesctl/cli/core_commands.py

```diff
@@ -30,7 +28,9 @@ def apply(ctx: click.Context, dry: bool, diff: bool, manifests_dir: str) -> None
     """
     config = ctx.obj["CONFIG"]
     taxonomy = _parse.parse(manifests_dir)
-    _apply.apply(
+    with_analytics(
```

Comment:
Do you foresee this being a decorator eventually that we can just add to the `core_commands.apply` function?

Reply:
I agree a decorator would be nice

Reply:
Yes I definitely want to refactor this into a decorator, but I struggled. My decorator chops are super weak. Any suggestions would be welcome.

Reply:
Created #393 to track a refactor for this.